src/reducers/gallery.js

```diff
@@ -0,0 +1,23 @@
+
+const initialGalleryState = [];
+const initialIsFetchingGallery = false
+export const gallery = (state = initialGalleryState, action) => {
+    switch (action.type) {
+        case 'FETCH_GALLERY_SUCCESS':
```

Comment:
create another file actionTypes.js
declare all string constants in that file
all other files should refer to those strings for action types